app/frontend/src/components/base/BaseDialog.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-dialog v-model="show" width="500">
```

Comment:
With a width of 500, have we tested this on very small screens to see if there's any potential overflow issues?

Reply:
good idea. this should be max-width="500"